www/front_src/src/Resources/Graph/Performance/Graph/index.tsx

```diff
@@ -466,6 +592,62 @@ const GraphContent = ({
                 />
               )}
             </Group>
+            {canNavigateInGraph && (
+              <>
```

Comment:
We might want to extract that to a dedicated Component. It might make sense to create a Context as well

Reply:
Why create a Context ? What is the purpose of this Context ?